lib/seahorn/Bmc.cc

```diff
@@ -216,6 +216,14 @@ namespace seahorn
     return m_model.eval (v, complete);
   }
   
+  DenseSet<const BasicBlock*> BmcTrace::getRegion()
```

Comment:
This does not look right. `m_bbs` is the sequence of all basic blocks in the bmc trace. 
There is no need to return them as a set. Moreover, passing sets by value in expensive.
If you need a set of bbs, traverse the trace and put them into the set in the client.